pkg/daemon/ceph/client/upgrade_test.go

```diff
@@ -124,7 +124,7 @@ func TestOkToContinue(t *testing.T) {
 	executor := &exectest.MockExecutor{}
 	context := &clusterd.Context{Executor: executor}
 
-	err := OkToContinue(context, "rook-ceph", "rook-ceph-mon-a", "mon", "a") // mon is not checked on ok-to-continue so nil is expected
+	err := OkToContinue(context, "rook-ceph", "rook-ceph-mon-a", "mon", "a", false) // mon is not checked on ok-to-continue so nil is expected
```

Comment:
What about a test for this case where it is not ok to continue? Then we can test the new var.